elements/lisk-cryptography/src/buffer.ts

```diff
@@ -37,20 +37,23 @@ export const intToBuffer = (
 				buffer.writeUIntBE(Number(value), 0, byteLength);
 			}
 		} else {
+			// eslint-disable-next-line no-lonely-if
 			if (signed) {
 				buffer.writeBigInt64BE(BigInt(value));
 			} else {
 				buffer.writeBigUInt64BE(BigInt(value));
 			}
 		}
 	} else {
+		// eslint-disable-next-line no-lonely-if
```

Comment:
At what point we will update the code based on new eslint rules. I suggest for such small things we should fix them right away.

Reply:
I was thinking to fix this, but it just made it hard to read, so i wasn't sure how to make it clearer

Reply:
We should then create a separate issue saying remove these comments and fix the code.

Reply:
this function will be removed with serialization =)